app/scripts/templates/settings/account_recovery/recovery_key.mustache

```diff
@@ -0,0 +1,32 @@
+<div id="account-recovery-key">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Recovery key{{/t}}</h2>
+
+        <p class="description">
+            {{#unsafeTranslate}}<span class="note">NOTE: </span>Keep your recovery key in a safe place. You will not be able to access this page again. You can download or print your key or email it to someone you trust{{/unsafeTranslate}}
+        </p>
+
+        <div class="recovery-key-container">
+            <div class="recovery-key">{{recoveryKey}}</div>
+        </div>
+
+        <div class="options">
+            <div class="title-container">
+                <div class="line">&boxh;&boxh;&boxh;&nbsp;</div>
+                <div class="title">{{#t}}Download your recovery key or print it{{/t}}</div>
+                <div class="line">&nbsp;&boxh;&boxh;&boxh;</div>
+            </div>
+
+            <div class="buttons">
+                <button class="download-key"><img alt="{{#t}}Download button image{{/t}}" class="recovery-icon-option" src="/images/recovery_key_download.svg">{{#t}}Download{{/t}}</button>
```

Comment:
I noticed that neither of these buttons have visible hover states, is that intentional? The cursor changes, but no color changes.

![button-hover](https://user-images.githubusercontent.com/848085/43536109-ab17429a-95b3-11e8-94e2-3ee9e898fb52.gif)

Reply:
Hmm not sure, but I'll flag for UX follow up in https://github.com/mozilla/fxa-content-server/issues/6394.